src/components/timeline/TrackScreenshots.js

```diff
@@ -140,72 +186,54 @@ export default explicitConnect<OwnProps, StateProps, DispatchProps>({
       trackHeight: getScreenshotTrackHeight(state),
     };
   },
+  mapDispatchToProps: {
+    updatePreviewSelection,
+  },
   component: withSize<Props>(Screenshots),
 });
 
 type HoverPreviewProps = {|
   +thread: Thread,
   +rangeStart: Milliseconds,
   +rangeEnd: Milliseconds,
-  +screenshots: Marker[],
   +isMakingPreviewSelection: boolean,
   +offsetX: null | number,
   +pageX: null | number,
   +containerTop: null | number,
   +width: number,
   +trackHeight: number,
+  +payload: null | ScreenshotPayload,
```

Comment:
I would even suggest to never take `null` (after all it makes no sense), and instead do the ternary in `Screenshots`:
```js
<div ...>
  <ScreenshotStrip .../>
  { payload ? <HoverPreview ...> : null }
</div>
```